src/AppBundle/Form/Admin/SubscriptionType.php

```diff
@@ -21,6 +21,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('ticketNo')
             ->add('contact', new ContactType(), array('by_reference' => false))
             ->add(
+                'environment',
+                'choice',
+                array(
+                    'choices'  => array('connect' => 'Connect', 'production' => 'Production'),
+                    'expanded' => true,
+                    'data' => 'connect',
```

Comment:
Nope: http://symfony.com/doc/2.0/reference/forms/types/field.html#data
Not that it matters, this can't be edited.